src/exceptions.php

```diff
@@ -29,3 +29,10 @@ class InvalidArgumentException extends LogicException {}
  * @author David Kudera <kudera.d@gmail.com>
  */
 class NotImplementedException extends LogicException {}
+
+
+/**
+ *
+ * @author Martin Janeček <admin@ikw.cz>
+ */
+class InvalidStateException extends \RuntimeException {}
```

Comment:
Could you please create new `RuntimeException` class and extend `InvalidStateException` from the new one?